api/Navigator.json

```diff
@@ -548,10 +548,24 @@
               "notes": "Not supported on macOS, see <a href='https://crbug.com/1144920'>bug 1144920</a>."
             },
             "firefox": {
-              "version_added": false
+              "version_added": "96",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "#dom.webshare.enabled",
```

Comment:
Could you remove the leading `#` and change `value_to_set` to `true`? Same for `share`.